core/package.json

```diff
@@ -76,7 +76,7 @@
   },
   "license": "MIT",
   "engines": {
-    "npm": ">=8.0.0 <9.0.0",
```

Comment:
I think we could remove the maximum npm version and just have the minimum, since all versions beginning with 9 should not be breaking changes with other 9 versions.
Alternatively we could set <10 instead